actioncable/app/assets/javascripts/action_cable/subscriptions.coffee

```diff
@@ -58,7 +58,7 @@ class ActionCable.Subscriptions
 
   sendCommand: (subscription, command) ->
     {identifier} = subscription
-    if identifier is ActionCable.INTERNAL.identifiers.ping
+    if identifier is ActionCable.INTERNAL.message_types.ping
```

Comment:
I think moving `ping` to a message type is still useful because it signifies that we are standardizing on message types as the only way to declare internal identifiers. I can understand how this could be interpreted as a cosmetic change, but I think it's an important design decision.